examples/dummy/apps/web/app/posts/ui/like-button.module.scss

```
@@ -1,11 +1,33 @@
 // Like control. Semantic tokens only, so the dark theme needs no second rule.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)